src/platform/site-wide/sass/fonts.scss

```diff
@@ -1,15 +1,13 @@
-$font-path: "~/src/site/assets/fonts";
-
 @font-face {
     font-family: "Source Sans Pro";
     font-style: normal;
     font-weight: 400;
-    src: url("../../../site/assets/fonts/sourcesanspro-regular-webfont.ttf") format("truetype");
+    src: url("/fonts/sourcesanspro-regular-webfont.ttf") format("truetype");
```

Comment:
I'm a bit dubious about this being the correct path, but when I used `./fonts/..` instead it cause an issue with the `verify-your-enrollment` application since it is importing the sass directly. I'm not knowledgable enough on how to deal with this situation so any advice is appreciated